### ISSUE TITLE
"Unhandled key-code [dec]: 8" on Backspace

### ISSUE THREAD

**Issue by Nuyube** [wren-lang/wren-cli#131]
Wren_CLI seems to fail whenever I press backspace in the window. (Windows 11 x64).

Reproduction:
- Download Wren_CLI 0.4.0 for Windows
- Run wren_cli.exe
- Press any key
- Attempt to erase your entry with backspace
- "Unhandled key-code [dec]: 8"

**Comment by Nuyube** [comment]
![wren_cli_u10M7tM8qE](https://user-images.githubusercontent.com/81436002/148493339-70749602-56d0-41c1-8ee1-43d36ee8dd9b.gif)
Here is a GIF of the problem.

**Comment by joshgoebel** [comment]
This has been fixed in `wren-console`, if you wanted to port it over:

https://github.com/joshgoebel/wren-console/commit/e03dfbcdba3e58817133f0350edcb272b72a8afe

**Comment by ruby0x1** [comment]
I don't see one but do you remember if there was a PR to fix that for wren-cli @joshgoebel?

**Comment by joshgoebel** [comment]
I don't think so, but I _think_ you could cherry-pick it right out of my repo most likely... I don't think that part of the source is diverged.

**Comment by ruby0x1** [comment]
I see, thanks
